docs/website-guidelines/kit-structure.md

```diff
@@ -25,49 +25,49 @@ Each folder/KIT's content is structured in at least four pages/subfolders:
     root
     └── docs-kits
         └── kits
-            └── NewKIT
+            └── new-kit
     ```
 
-2. Inside the `NewKIT` folder, add a `page_adoption-view.md` file with the content that suits your use case
+2. Inside the `new-kit` folder, add a `page_adoption-view.md` file with the content that suits your use case
 
     ```md
     root
     └── docs-kits
         └── kits
-            └── NewKIT
+            └── new-kit
                 └── page_adoption-view.md
     ```
 
-3. Inside the `NewKIT` folder, add a `page_software-operation-view.md` file with the content that suits your use case
+3. Inside the `new-kit` folder, add a `page_software-operation-view.md` file with the content that suits your use case
 
     ```md
     root
     └── docs-kits
         └── kits
-            └── NewKIT
+            └── new-kit
                 ├── page_adoption-view.md
                 └── page_software-operation-view.md
     ```
 
-4. Inside the `NewKIT` folder, add a `page_documentation.md` file with the content that suits your use case
+4. Inside the `new-kit` folder, add a `page_documentation.md` file with the content that suits your use case
 
     ```md
     root
     └── docs-kits
         └── kits
-            └── NewKIT
+            └── new-kit
                 ├── page_adoption-view.md
                 ├── page_software-operation-view.md
                 └── page_documentation.md
     ```
 
-5. Inside the `NewKIT` folder, add a `SoftwareDevelopmentView` folder that contains a `page_software-development-view.md` file with the content that suits your use case
+5. Inside the `new-kit` folder, add a `SoftwareDevelopmentView` folder that contains a `page_software-development-view.md` file with the content that suits your use case
```

Comment:
Why do we describe to add a `SoftwareDevelopmentView` (shouldnt this also be kebab-case?) but in the example the folder is named `Software Development View`